mapswipe_workers/requirements.txt

```diff
@@ -4,12 +4,12 @@ click==8.1.3
 firebase-admin==4.4.0
 flake8==3.8.3
 mapswipe-workers==3.0
-pandas==1.4.3
+pandas==1.1.5
```

Comment:
Why are you downgrading `pandas`?

Reply:
worker docker build was failing due to pandas==1.4.3. Which was solved by downgrading pandas.